src/Tribe/Customizer/Global_Elements.php

```diff
@@ -39,21 +39,53 @@ public function get_css_template( $template ) {
 			$accent_color_rgb = $accent_color::hexToRgb( $settings['accent_color'] );
 			$accent_css_rgb   = $accent_color_rgb['R'] . ',' . $accent_color_rgb['G'] . ',' . $accent_color_rgb['B'];
 
+			$accent_color_hover          = 'rgba(' . $accent_css_rgb . ',0.8)';
+			$accent_color_active         = 'rgba(' . $accent_css_rgb . ',0.9)';
+			$accent_color_background     = 'rgba(' . $accent_css_rgb . ',0.07)';
+			$accent_color_multiday       = 'rgba(' . $accent_css_rgb . ',0.24)';
+			$accent_color_multiday_hover = 'rgba(' . $accent_css_rgb . ',0.34)';
+			$color_background            = '#ffffff';
+
+			// overrides for common base/full/forms/_toggles.pcss
+			$template .= '
+				.tribe-common .tribe-common-form-control-toggle__input:checked {
+					background-color: <%= global_elements.accent_color %>;
+				}
+			';
+
+			// overrides for common base/full/typography/_ctas.pcss
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: d00989e8c990d6fd45ec134c8db6b0e7</sub>